cmd/zpool/zpool_main.c

```diff
@@ -2173,19 +2173,84 @@ show_import(nvlist_t *config)
 	}
 }
 
+typedef long long unsigned int llu;
+
+static void
+extract_timestamp_and_txg(nvlist_t *config, uint64_t *txg,
+    uint64_t *timestamp)
+{
+	verify(nvlist_lookup_uint64(config, ZPOOL_CONFIG_TIMESTAMP,
+	    timestamp) == 0);
+	verify(nvlist_lookup_uint64(config, ZPOOL_CONFIG_FOUND_TXG,
+	    txg) == 0);
+}
+
+static int
+check_pool_for_activity(libzfs_handle_t *hdl, nvlist_t *config)
+{
+	nvlist_t *nvl = NULL;
+	uint64_t original_txg, original_timestamp;
+	int rc = 0;
+	char *poolname;
+	int seconds_between_checks = 1;
+	int times_to_check = 2;
```

Comment:
Yes.  I keep monkeying with these times to make testing easier, depending on what piece I'm changing ATM.